workflow/controller/indexes/labels.go

```diff
@@ -27,6 +28,20 @@ func MetaWorkflowPhaseIndexFunc() cache.IndexFunc {
 	}
 }
 
+func MetaPodPhaseIndexFunc() cache.IndexFunc {
```

Comment:
I think this
*  should be in it's own file - nothing to do with labels
* be implemented like indexes.MetaWorkflowIndexFunc (i.e. no need for second-order function)
* named `PodPhaseIndexFunc` as it does not use `meta` like other funcs

Reply:
Done